09:58:55 18025 Wrk-0.2 | *** Break ***: segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00007f3d1ddd5687 in __GI___waitpid (pid=18528, stat_loc=stat_loc
entry=0x7ffea29d10e8, options=options
entry=0) at ../sysdeps/unix/sysv/linux/waitpid.c:30
#1  0x00007f3d1dd40067 in do_system (line=<optimised out>) at ../sysdeps/posix/system.c:149
#2  0x00007f3d1ebec05b in TUnixSystem::Exec (shellcmd=<optimised out>, this=0x5635da0d47c0) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:2119
#3  TUnixSystem::StackTrace (this=0x5635da0d47c0) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:2413
#4  0x00007f3d1ebeea44 in TUnixSystem::DispatchSignals (this=0x5635da0d47c0, sig=kSigSegmentationViolation) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:3644
#5  <signal handler called>
#6  __memmove_avx_unaligned_erms () at ../sysdeps/x86_64/multiarch/memmove-vec-unaligned-erms.S:423
#7  0x00007f3d1dd7c994 in _IO_new_file_xsputn (f=0x7f3d1e0dd760 <_IO_2_1_stdout_>, data=<optimised out>, n=1936279649) at fileops.c:1258
#8  0x00007f3d1dd70977 in __GI__IO_fwrite (buf=0x333937362e74, size=1, count=1936279649, fp=0x7f3d1e0dd760 <_IO_2_1_stdout_>) at iofwrite.c:39
#9  0x00007f3d1e62ca92 in std::ostream::write(char const*, long) () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#10 0x00007f3d1ea97b21 in operator<< (os=..., s=...) at /work/root/v6.14.04/core/base/src/Stringio.cxx:182
#11 0x00007f3d0b25630a in HS::FIT::FitManager::FillEventsPDFs(unsigned int) () from /home/dglazier/.proof/cache/FitManager_C.so
#12 0x00007f3d0b257a3a in HS::FIT::FitManager::Run(unsigned int) () from /home/dglazier/.proof/cache/FitManager_C.so
#13 0x00007f3d0ac9a3da in HS::FIT::sPlot::Run(unsigned int) () from /home/dglazier/.proof/cache/sPlot_C.so
#14 0x00007f3d0b2574b1 in HS::FIT::FitManager::RunOne(int) () from /home/dglazier/.proof/cache/FitManager_C.so
#15 0x00007f3d0aa89d77 in HS::FIT::FitSelector::Process(long long) () from /home/dglazier/.proof/cache/FitSelector_C.so
#16 0x00007f3d0a63a567 in TProofPlayer::Process (this=0x5635e0fba910, dset=0x5635e0e2eba0, selector_file=0x0, option=0x7ffea29d49e9 "", nentries=-1, first=-1) at /work/root/v6.14.04/proof/proofplayer/src/TProofPlayer.cxx:1314
#17 0x00007f3d1140c979 in TProofServ::HandleProcess (this=0x5635dad6c820, mess=<optimised out>, slb=0x0) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:4047
#18 0x00007f3d11407ef7 in TProofServ::HandleSocketInput (this=0x5635dad6c820, mess=0x5635e02d6000, all=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:1653
#19 0x00007f3d113fb142 in TProofServ::HandleSocketInput (this=0x5635dad6c820) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:1364
#20 0x00007f3d11411fd7 in TProofServLiteInputHandler::Notify (this=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServLite.cxx:177
#21 TProofServLiteInputHandler::ReadNotify (this=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServLite.cxx:169
#22 0x00007f3d1ebede30 in TUnixSystem::CheckDescriptors (this=this
entry=0x5635da0d47c0) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:1322
#23 0x00007f3d1ebef7c8 in TUnixSystem::DispatchOneEvent (this=0x5635da0d47c0, pendingOnly=<optimised out>) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:1077
#24 0x00007f3d1eb05271 in TSystem::InnerLoop (this=0x5635da0d47c0) at /work/root/v6.14.04/core/base/src/TSystem.cxx:411
#25 TSystem::Run (this=0x5635da0d47c0) at /work/root/v6.14.04/core/base/src/TSystem.cxx:361
#26 0x00007f3d1ea97def in TApplication::Run (this=0x5635dad6c820, retrn=<optimised out>) at /work/root/v6.14.04/core/base/src/TApplication.cxx:1173
#27 0x00005635d80cb660 in main (argc=<optimised out>, argv=0x7ffea29d5de8) at /work/root/v6.14.04/main/src/pmain.cxx:325
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#6  __memmove_avx_unaligned_erms () at ../sysdeps/x86_64/multiarch/memmove-vec-unaligned-erms.S:423
#7  0x00007f3d1dd7c994 in _IO_new_file_xsputn (f=0x7f3d1e0dd760 <_IO_2_1_stdout_>, data=<optimised out>, n=1936279649) at fileops.c:1258
#8  0x00007f3d1dd70977 in __GI__IO_fwrite (buf=0x333937362e74, size=1, count=1936279649, fp=0x7f3d1e0dd760 <_IO_2_1_stdout_>) at iofwrite.c:39
#9  0x00007f3d1e62ca92 in std::ostream::write(char const*, long) () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#10 0x00007f3d1ea97b21 in operator<< (os=..., s=...) at /work/root/v6.14.04/core/base/src/Stringio.cxx:182
#11 0x00007f3d0b25630a in HS::FIT::FitManager::FillEventsPDFs(unsigned int) () from /home/dglazier/.proof/cache/FitManager_C.so
#12 0x00007f3d0b257a3a in HS::FIT::FitManager::Run(unsigned int) () from /home/dglazier/.proof/cache/FitManager_C.so
#13 0x00007f3d0ac9a3da in HS::FIT::sPlot::Run(unsigned int) () from /home/dglazier/.proof/cache/sPlot_C.so
#14 0x00007f3d0b2574b1 in HS::FIT::FitManager::RunOne(int) () from /home/dglazier/.proof/cache/FitManager_C.so
#15 0x00007f3d0aa89d77 in HS::FIT::FitSelector::Process(long long) () from /home/dglazier/.proof/cache/FitSelector_C.so
#16 0x00007f3d0a63a567 in TProofPlayer::Process (this=0x5635e0fba910, dset=0x5635e0e2eba0, selector_file=0x0, option=0x7ffea29d49e9 "", nentries=-1, first=-1) at /work/root/v6.14.04/proof/proofplayer/src/TProofPlayer.cxx:1314
#17 0x00007f3d1140c979 in TProofServ::HandleProcess (this=0x5635dad6c820, mess=<optimised out>, slb=0x0) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:4047
#18 0x00007f3d11407ef7 in TProofServ::HandleSocketInput (this=0x5635dad6c820, mess=0x5635e02d6000, all=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:1653
#19 0x00007f3d113fb142 in TProofServ::HandleSocketInput (this=0x5635dad6c820) at /work/root/v6.14.04/proof/proof/src/TProofServ.cxx:1364
#20 0x00007f3d11411fd7 in TProofServLiteInputHandler::Notify (this=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServLite.cxx:177
#21 TProofServLiteInputHandler::ReadNotify (this=<optimised out>) at /work/root/v6.14.04/proof/proof/src/TProofServLite.cxx:169
#22 0x00007f3d1ebede30 in TUnixSystem::CheckDescriptors (this=this
entry=0x5635da0d47c0) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:1322
#23 0x00007f3d1ebef7c8 in TUnixSystem::DispatchOneEvent (this=0x5635da0d47c0, pendingOnly=<optimised out>) at /work/root/v6.14.04/core/unix/src/TUnixSystem.cxx:1077
#24 0x00007f3d1eb05271 in TSystem::InnerLoop (this=0x5635da0d47c0) at /work/root/v6.14.04/core/base/src/TSystem.cxx:411
#25 TSystem::Run (this=0x5635da0d47c0) at /work/root/v6.14.04/core/base/src/TSystem.cxx:361
#26 0x00007f3d1ea97def in TApplication::Run (this=0x5635dad6c820, retrn=<optimised out>) at /work/root/v6.14.04/core/base/src/TApplication.cxx:1173
#27 0x00005635d80cb660 in main (argc=<optimised out>, argv=0x7ffea29d5de8) at /work/root/v6.14.04/main/src/pmain.cxx:325
===========================================================


09:59:02 18025 Wrk-0.2 | Error in <TProofServLite::HandleException>: caugth exception triggered by signal '1' check logs for possible stacktrace - last cycle: 6
